CONTRIBUTING.md

```diff
@@ -100,7 +101,11 @@ Sometimes you only need to add a new route to an existing resource. You will nee
 
 The response to that route will go into that resources `responses/` folder.
 
-You then want to make sure that you add the new path in `Lob-API-public.yml` that will reference your new file, making sure that more specific paths happen first in the order they are listed..
+You then want to make sure that you add the new path in `Lob-API-public.yml` that will reference your new file, making sure that more specific paths happen first in the order they are listed.
+
+You can add the tests for your route to the existing test file for the resource, or create a separate test file for your route. The choice is up to you. Some factors
+to take into consideration include the length and complexity of the existing test file and your new tests and how much of the code in existing test you might
+be able to reuse. _If you do put your test in a separate file, make sure you modify the entry for your resource in `config: goalie-mappings` in `package.json` so that your tests get run during CI and monitoring!_
 
 ## Documentation
```

Comment:
please remove, i erroneously left it in from a previous organization of this file in my pr